sdk/test/AzureTest/DVRSessionTransport.swift

```diff
@@ -64,6 +64,9 @@ public class DVRSessionTransport: TransportStage {
             .appendingPathComponent("sdk/communication/AzureCommunicationChat/Tests/Recordings").absoluteString else {
             fatalError("SDK Path Invalid")
         }
+        let replacements = ["authorization": "", "client-request-id" : "", "retry-after" : "","x-ms-client-request-id": "","x-ms-correlation-request-id": "", "x-ms-ratelimit-remaining-subscription-reads": "",
+                            "x-ms-request-id":"","x-ms-routing-request-id":"","x-ms-gateway-service-instanceid":"","x-ms-ratelimit-remaining-tenant-reads":"","x-ms-served-by":"","x-ms-authorization-auxiliary":""]
+        
```

Comment:
So in python, this would simply be a list of keys that would be passed in as a header filter and handled by DVR.

Reply:
With this PR (https://github.com/tjprescott/DVR/pull/5) you'd be able to omit these with something like:
```swift
let filters = ["authorization": .remove, "client-request-id": .remove, ...]
```

For now, instead of leveraging `replacements`, I think you should just drop these headers in the `beforeRecordRequest` closure.

Reply:
This was completed in #935.